api/requirements.txt

```diff
@@ -5,7 +5,7 @@ blinker==1.6.2
 cachetools==5.3.1
 certifi==2023.7.22
 click==8.1.3
-cryptography==41.0.3
+cryptography
```

Comment:
Why Rolling back in this ?

Reply:
When adding msal it complains of conflicting dependency